### ISSUE TITLE
Space and lasso tool not moving groups and text annotations inside a Collaboration

### ISSUE THREAD

**Issue by fatimagarmar** [bpmn-io/bpmn-js#2226]
### Describe the Bug

When using the Space Tool or the Lasso Tool in bpmn-js version 17.9.2, the tools do not move groups or text annotations when clicking inside a bpmn:Collaboration. However, all other elements are moved correctly. This issue appears to be specific to elements within a Collaboration.

### Steps to Reproduce

1. Open a BPMN diagram in the editor.
2. Add a bpmn:Collaboration with various elements, including groups and text annotations.
3. Select the Space Tool or Lasso Tool.
4. Click and drag to create space or select elements inside the Collaboration.
5. Observe that groups and text annotations are not moved, while other elements are moved as expected.

### Expected Behavior

All elements, including groups and text annotations, should be moved when using the Space Tool or Lasso Tool.

### Environment

 - Browser: Chrome Version 126.0.6478.127
 - OS: Windows 11 Pro
 - Library version: bpmn-js 17.9.2

![bug space tool](https://github.com/user-attachments/assets/f8232dd0-a1a1-4726-9250-9e43aa05d6a0)

---

Related to https://github.com/camunda/camunda-modeler/issues/4797

**Comment by jarekdanielak** [comment]
The text annotation will not move with other elements unless it's also selected with the lasso. Same way it won't move if you move its parent element. It might be a problem when using lasso inside of collaboration. You can try to select elements inside the collaboration with lasso tool and shift click the annotation to move it all together.

As for the group, I do indeed see some inconsistencies when trying to lasso it, but could you please share a screen recording of your exact incorrect behavior so that we are on the same page?

**Comment by fagarma** [comment]
> The text annotation will not move with other elements unless it's also selected with the lasso. Same way it won't move if you move its parent element. It might be a problem when using lasso inside of collaboration. You can try to select elements inside the collaboration with lasso tool and shift click the annotation to move it all together.
> 
> As for the group, I do indeed see some inconsistencies when trying to lasso it, but could you please share a screen recording of your exact incorrect behavior so that we are on the same page?


https://github.com/user-attachments/assets/b25cbd94-9de8-4d7a-b5ea-d53687f6f456


If I'm not mistaken the space tool should move everything inside the collaboration, but it won't move text annotations or groups.

**Comment by jarekdanielak** [comment]
Space tool is local by default. Try using it while holding Shift key.

**Comment by nikku** [comment]
I tend to agree with @fagarma that the desired behavior would be for text annotations (they live outside of the process) _should_ move with the annotated target. 

@fagarma Could you check if https://github.com/camunda/camunda-modeler/issues/4172, reported in an app using bpmn-js is the same error?

**Comment by fagarma** [comment]
> I tend to agree with @fagarma that the desired behavior would be for text annotations (they live outside of the process) _should_ move with the annotated target.
> 
> @fagarma Could you check if [camunda/camunda-modeler#4172](https://github.com/camunda/camunda-modeler/issues/4172), reported in an app using bpmn-js is the same error?

Yes, it is the same issue as reported there

**Comment by nikku** [comment]
Moving to backlog, awaiting further user feedback.

**Comment by fagarma** [comment]
> Moving to backlog, awaiting further user feedback.

Sorry, do you need anything more from me?

**Comment by nikku** [comment]
> > Moving to backlog, awaiting further user feedback.
>
> Sorry, do you need anything more from me?> Sorry, do you need anything more from me?

I just realized how my previous comment was very confusing. My appologies.

We don't need additional information from you, but seek validation from other users, too. Thank you, again, for the report. :medal_sports:

**Comment by barmac** [comment]
This was also reported via support, cf. https://github.com/camunda/camunda-modeler/issues/4797

**Comment by jarekdanielak** [comment]
Closed via https://github.com/bpmn-io/bpmn-js/pull/2285
